odoo/exceptions.py

```diff
@@ -63,6 +63,13 @@ def __init__(self, msg):
         super(AccessError, self).__init__(msg)
 
 
+class CacheMissingError(except_orm, KeyError):
```

Comment:
I would rename this to `CacheMissException`, currently `CacheMissing` implies that the cache is missing, not that there was a cache miss, which is weird.